x/concentrated-liquidity/lp_test.go

```diff
@@ -585,6 +614,8 @@ func (s *KeeperTestSuite) TestWithdrawPosition() {
 			// Dumb sanity-check that creating a position with the same liquidity amount after fully removing it does not error.
 			// This is to be more thoroughly tested separately.
 			if expectedRemainingLiquidity.IsZero() {
+				// Add one because we withdraw one less than originally fonded due to truncation in favor of the pool.
+				s.FundAcc(owner, sdk.NewCoins(sdk.NewCoin(USDC, config.amount0Desired)))
```

Comment:
I don't see one added here

Reply:
Meant to be `sdk.OneInt()` instead of `config.amount0Desired`. Changed now